priv/rpc/ipfs/rpc/file.yml

```diff
@@ -19,7 +19,7 @@ public:
     desc: flush a given path's data to disk.
     method: files/flush
     args: ["arg"]
-  - name: ls
+  - name: files_ls
```

Comment:
same name and args with ls (file/ls), so change ls to files_ls in files/ls

Reply:
great findings! can you help to change preivous `ls` to `unix_ls` and keep this as it is?